edgecast/waf/access_rule.go

```diff
@@ -2,7 +2,16 @@
 
 package waf
 
-import "fmt"
+import (
+	"fmt"
+)
+
+type TrustedEntities struct {
```

Comment:
`TrustedEntities`is not needed - this is what the `AccessControls` struct covers